src/boot/efi/secure-boot.c

```diff
@@ -33,7 +33,7 @@ SecureBootMode secure_boot_mode(void) {
 }
 
 /*
- * Custom mode allows to change secure boot certificate databases db, dbx, KEK and PK without the variable
+ * Custom mode allows one to change secure boot certificate databases db, dbx, KEK and PK without the variable
```

Comment:
Custom mode allows the  secure boot certificate databases db, dbx, KEK, and PK to be changed …"